CMake/SlicerApplicationOptions.cmake

```diff
@@ -120,23 +120,33 @@ mark_as_superbuild(${Slicer_MAIN_PROJECT_APPLICATION_NAME}_SOURCE_DIR)
 #-----------------------------------------------------------------------------
 if(WIN32)
 
+  if(NOT DEFINED Slicer_CPACK_NSIS_INSTALL_REQUIRES_ADMIN_ACCOUNT)
+    set(Slicer_CPACK_NSIS_INSTALL_REQUIRES_ADMIN_ACCOUNT OFF CACHE BOOL
+      "Require administrator account to install the application. Must be enabled if Slicer_CPACK_NSIS_INSTALL_ROOT is only writable by administrators.")
+    mark_as_advanced(Slicer_CPACK_NSIS_INSTALL_REQUIRES_ADMIN_ACCOUNT)
+  endif()
+  mark_as_superbuild(Slicer_CPACK_NSIS_INSTALL_REQUIRES_ADMIN_ACCOUNT:BOOL)
+  message(STATUS "Configuring ${Slicer_MAIN_PROJECT_APPLICATION_NAME} requires admin account [${Slicer_CPACK_NSIS_INSTALL_REQUIRES_ADMIN_ACCOUNT}]")
+
   if(NOT DEFINED Slicer_CPACK_NSIS_INSTALL_ROOT)
     # Set root install directory (displayed to end user at installer-run time)
     # to C:\Users\<username>\AppData\Local by default to allow installation
     # without having administrative privileges.
-    set(Slicer_CPACK_NSIS_INSTALL_ROOT "$LOCALAPPDATA\\\\${Slicer_ORGANIZATION_NAME}" CACHE STRING
+    set(_install_root_default "$LOCALAPPDATA\\\\${Slicer_ORGANIZATION_NAME}")
```

Comment:
Looks good. It would be a bit more readable like this:

    # Set default install directory (displayed to end user during installation)
     if(Slicer_CPACK_NSIS_INSTALL_REQUIRES_ADMIN_ACCOUNT)
      # User has administrative privileges, therefore we can install to shared folder
      # "C:\Program Files" or "c:\Program Files (x86)".
      if(CMAKE_CL_64)
        set(_install_root_default "$PROGRAMFILES64")
      else()
        set(_install_root_default "$PROGRAMFILES")
      endif()
    else()
      # We do not require administrative privileges, therefore we install to user folder
      # "C:\Users\<username>\AppData\Local".
      set(_install_root_default "$LOCALAPPDATA\\\\${Slicer_ORGANIZATION_NAME}")
    endif()